opds_import.py

```diff
@@ -844,7 +848,8 @@ def consolidate_links(cls, links):
 
         Similarly if link n is a thumbnail and link n+1 is an image.
         """
-        new_links = list(links)
+        new_links = [x for x in links if x]
+        links = list(new_links)
```

Comment:
That makes sense. It would probably be clearer looking at all the code instead of this diff, but maybe just add a comment.